spacy/tests/training/test_training.py

```diff
@@ -344,7 +344,7 @@ def test_gold_biluo_4791(en_vocab, en_tokenizer):
         doc, {"words": gold_words, "spaces": gold_spaces, "entities": entities}
     )
     ner_tags = example.get_aligned_ner()
-    assert ner_tags == ["O", "O", "O", "O", "U-MONEY", "O"]
+    assert ner_tags == ["O", "O", "O", "O", "B-MONEY", "L-MONEY", "O"]
```

Comment:
This test is about alignment, not tokenization, so this makes it equivalent to the following `$` case rather than testing the alignment. I think a better change would be to use `A` instead of `₹` or something else that results in one token with the default tokenizer. Or the other option is to use a custom tokenizer rather than `en_tokenizer` that treats `₹` as before. Or I guess this was probably more of an issue with `GoldParse`, and I guess a third option is to discard the test entirely.

Reply:
Ah, thanks for explaining that. It looks like changing the rupee symbol to A works, so I'll do that, but since this is GoldParse related maybe the easiest thing is to just delete it?

Reply:
I think we can leave it for now. Overall the test suite could use some spring cleaning because it's starting to be too slow, but I think this is fine for now.